grunt_tasks/fontello.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Define tasks that download and copy fontello font files.
+ */
+module.exports = function (grunt) {
+    var path = require('path');
+
+    grunt.config.merge({
+        curl: {
+            fontello: {
+                src: 'https://data.kitware.com/api/v1/file/57c5d1fc8d777f10f269dece/download',
```

Comment:
I considered it, but it's around a 0.5MB diff every time we want to add or remove an icon.  I don't care much either way though.